docs/features/backup_and_restore.md

```diff
@@ -6,7 +6,201 @@ hide_title: false
+Most of your Cumulus deployment can be recovered by redeploying via Terraform.
+However, the Cumulus metadata including providers, collections, granules, rules,
+and executions that are stored in [DynamoDB](./data_in_dynamodb), and
+concurrently being written to the Core Postgres instance can only be
+restored if backup was properly configured or enabled. If a deployment is lost,
+logs and Step Function executions in the AWS console will be irrecoverable.
+
+## Postgres Database
+
+### Please note
+
+* Cumulus supports a "bring your own" Postgres instance approach, however
+  our reference implementation utilizes a serverless Aurora/RDS database - as
+  such this reference provides AWS RDS Aurora Serverless backup options.
+
+* As of Cumulus version 3, data storage is transitioning from the use of a
+  DynamoDB backed primary database to a Postgres instance with a reference Aurora/RDS Postgres
+  compatible module.     In this mode, the primary data records are still being
+  written to DynamoDB (and replicated to Elastic Search), however replicated
+  writes are being written to the required Postgres instance.  As such, the
+  primary source of recovery should be considered to be a re-migration of data
+  from the primary database.
+
+### Backup and Restore
+
+#### Re-migration
+
+In Cumulus version 3 the Postgres database is not the primary data store - as
+such, the simplest recovery approach is to re-deploy your database instance
+(e.g. your RDS cluster) and re-run the database data migration
+module to migrate Collections, Providers, etc, from a (possibly recovered
+DynamoDB instance) then begin active migration of Files and Granules records.
+
+For larger datastores this approach may not work
+due to migration time/other limitations.  In that case, utilizing an RDS recovery
+approach in conjunction with the Dynamo to RDS reconciliation tooling
+may be appropriate.
+
+#### Backup and Restore with AWS RDS
+
+##### Configuring Database Backups
+
+For AWS RDS Aurora database deployments, AWS provides a host of database
+backup/integrity options, including [PITR (Point In Time
+Recovery)](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_PIT.html)
+based on automated database backups and replay of transaction logs.
+
+For further information on RDS backup procedures, see the [AWS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_CommonTasks.BackupRestore.html)
+
+##### Disaster Recovery
+
+To recover a Cumulus Postgres database in a disaster or data-loss scenario, you should perform the following steps:
+
+* If the Postgres database cluster exists/is still online, halt workflow
+  activity, then take the cluster offline/remove access.
+* If needed, recover the DynamoDB tables as noted in the
+  [DynamoDb](./backup_and_restore#dynamodb) section of this document.
+* Redeploy a new database cluster from your backup, matching as closely as possible to the DynamoDB
+  restore time.   See [AWS's
```

Comment:
```
as closely as possible to the DynamoDB restore time
```
Is there a recommendation in terms of whether it is better to be behind or ahead of the DynamoDB time here?

Reply:
I don't know that i had that when this was being written.... 🤔 

Given our conversation yesterday, I would assume that we're going to want RDS to be *behind* the Dynamo time, as migration would pick up the difference.

Reply:
3aed94f76